### PR TITLE
PR for frontend tasks.

### PR DESCRIPTION
**Pull request by yogeshmpawar**
Added frontend tasks.

### CODE REVIEW

File: web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css [hunk 1]
```diff
@@ -7,3 +7,45 @@
 .toolbar-icon-style-guide::before {
   background-image: url(../assets/style_guide.svg);
 }
+
+.avatar {
+  background-image: url(../assets/Avatar.png);
+}
+
+.avatar-cody {
```
Comment (amitaibu) [inline web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css:15]: Shouldn't this be coming from the content? That's hardocding it to the css

File: web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css [hunk 2]
```diff
@@ -7,3 +7,45 @@
...
+
+.avatar-theresa {
+  background-image: url(../assets/Avatar-theresa.png);
+}
+
+.avatar-cameron {
+  background-image: url(../assets/Avatar-cameron.png);
+}
+
+.person-badge {
```
Comment (amitaibu) [inline web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css:43]: Please use TW

File: web/modules/custom/server_style_guide/templates/server-style-guide-person-card.html.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<div style="width: 344px; height: 100%; background: white; box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.06); border-radius: 8px; overflow: hidden; flex-direction: column; justify-content: flex-start; align-items: flex-start; display: inline-flex">
```
Comment (amitaibu) [inline web/modules/custom/server_style_guide/templates/server-style-guide-person-card.html.twig:1]: Please use TW syntax, here and below

**Review by amitaibu — COMMENTED** [review]
Thanks. A few comments

File: web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css (hunk 1 above)
Comment (yogeshmpawar) [reply web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css:15]: Addressed these feedbacks in - https://github.com/yogeshmpawar/pevb_task/pull/4 PR

File: web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css (hunk 2 above)
Comment (yogeshmpawar) [reply web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css:43]: Addressed these feedbacks in - https://github.com/yogeshmpawar/pevb_task/pull/4 PR

File: web/modules/custom/server_style_guide/templates/server-style-guide-person-card.html.twig (hunk 3 above)
Comment (yogeshmpawar) [reply web/modules/custom/server_style_guide/templates/server-style-guide-person-card.html.twig:1]: Addressed these feedbacks in - https://github.com/yogeshmpawar/pevb_task/pull/4 PR

**Comment by yogeshmpawar** [comment]
Addressed above feedbacks in - https://github.com/yogeshmpawar/pevb_task/pull/4 PR

**Pull request merged**